### PR TITLE
updated /en/regular-expression.html

### PR DESCRIPTION
**Pull request by myselfprincee**
### Added a real Life Pincode Checking Example.

### CODE REVIEW

**Comment by myselfprincee** [comment]
@sumn2u  Please have a look. I am looking to make more changes soon and improve the overall Website to make it better.

File: en/regular-expression.md [hunk 1]
```diff
@@ -34,17 +34,142 @@ In cases where we want to have a forward slash to be a part of a regular express
...
+const match = regex.test(str);
+// The .test() method returns a boolean value: 
+// true if the match is found, and false if the match is not found.
+
+console.log(match); // true
+```
+
+* `m` - multiline matching 
+
+Example:
```
Comment (sumn2u) [inline en/regular-expression.md]: Is it possible to put a space after the Example? Like you have done to the rest?

Comment (myselfprincee) [reply en/regular-expression.md:66]: @sumn2u in multiline matching ?

Comment (myselfprincee) [reply en/regular-expression.md]: image

is this what you looking for ??

a space between examples ?? @sumn2u

Comment (myselfprincee) [reply en/regular-expression.md]: or should we go for this 

image

**Comment by myselfprincee** [comment]
@sumn2u  Please reply

**Comment by sumn2u** [comment]
> Example:

Sorry for the confusion. Was asking if its possible to do like this `Example :`.

**Comment by myselfprincee** [comment]
> > Example:
> 
> Sorry for the confusion. Was asking if its possible to do like this `Example :`.

Yeah sure Just getting back with the solution in 4-5 minutes.

**Comment by myselfprincee** [comment]
@sumn2u  Done 👍

**Review by sumn2u — APPROVED** [review]
LGTM

**Pull request merged**